script.js

```diff
@@ -44,6 +44,21 @@ idApp.eventListeners = function() {
       }
    })
 
+   // Event Listener #3A for when any radio input is selected to display a div with some more info
+   $main.on('click', '.optionContainer', function () {
+      $('.hiddenContents').removeClass('displayContents');
+      $('.fa-plus').removeClass('active')
+      $(this).find('.hiddenContents').toggleClass('displayContents');
+      $(this).parent('.windowBLabel').find('.fa-plus').addClass('active')
+   })
+
+   // Event Listener #3B to hide the extra info by clicking the X button
+   $main.on('click', '.fa-plus', function() {
+      $(this).toggleClass('active');
+      $(this).siblings('.optionContainer').find('.hiddenContents').toggleClass('displayContents')
```

Comment:
Cool use of .siblings()! Hadn't heard of this method before, but I'm 👀 at the docs, and it sounds super useful!